src/main/java/seedu/address/storage/JsonModulePlanStorage.java

```diff
@@ -1,80 +1,79 @@
 package seedu.address.storage;
 
-import static java.util.Objects.requireNonNull;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.exceptions.DataLoadingException;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.commons.util.FileUtil;
+import seedu.address.commons.util.JsonUtil;
+import seedu.address.model.moduleplan.ReadOnlyModulePlan;
 
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.Optional;
 import java.util.logging.Logger;
 
-import seedu.address.commons.core.LogsCenter;
-import seedu.address.commons.exceptions.DataLoadingException;
-import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.commons.util.FileUtil;
-import seedu.address.commons.util.JsonUtil;
-import seedu.address.model.ReadOnlyAddressBook;
+import static java.util.Objects.requireNonNull;
 
```

Comment:
Seems to be the incorrect import ordering. The static imports should be followed by the standard java package, before the rest of the imports.

Perhaps you could run checkstylemain on gradle to verify that you have the correct import orderings.